packages/marble-api/scripts/openapi.yaml

```diff
@@ -116,40 +116,44 @@ paths:
             items:
               type: string
               format: uuid
-        - name: triggerObject[]
+        - name: trigger_object[]
           description: trigger objects used to filter the list
           in: query
           required: false
           schema:
             type: array
             items:
               type: string
-        - name: startDate
-          description: start date used to filter the list
+        - name: has_case
+          description: 'Optional filter: the decision is attached to a case'
```

Comment:
```suggestion
          description: 'Optional filter: the decision is attached to a case (true/false)'
```

Reply:
Good catch, I propose an alternative description (no filter behaviour is explicit) and copy paste the same description on the `public-api.yaml`